doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -392,8 +392,9 @@ Despite the old encoding will be deprecated in the future, it is possible to swi
 #### <a name="section2.3.4"></a>About supported versions of MongoDB
 This sink has been tested with the following versions of Mongo:
 
-* 3.2.6
```

Comment:
3.2.6 has been removed as it has passed End-of-Life according https://www.mongodb.com/support-policy